pkg/cmd/server/start/start_node.go

```diff
@@ -82,6 +89,8 @@ func NewCommandStartNode(basename string, out, errout io.Writer) (*cobra.Command
 	BindKubeConnectionArgs(options.NodeArgs.KubeConnectionArgs, flags, "")
 
 	flags.BoolVar(&options.NodeArgs.Bootstrap, "bootstrap", false, "Use the provided .kubeconfig file to perform initial node setup (experimental).")
```

Comment:
This now unnecessary. If you specify a bootstrap-config-name, this is true.  If you don't, this is false.

Reply:
Remove?

Reply:
No, you can bootstrap and use the default.

Reply:
> No, you can bootstrap and use the default.

I don't see why I'd do that.  I'd make them specify it and eliminate this flag.